### PR TITLE
Splash page

### PR DESCRIPTION
**Pull request by benborgers**
Put together a first iteration of a simple splash page. The design is based off of Kunj’s designs in Figma, and it’s pretty bold (although I like it) so let me know what you think.

### CODE REVIEW

File: docs/pages/_app.tsx [hunk 1]
```diff
@@ -60,11 +60,21 @@ export default function App({ Component, pageProps }: AppProps) {
     setNavOpen(false);
   }, [router.pathname]);
 
+  if (router.pathname === "/") {
+    return (
+      <>
+        <Head>
+          <title>NFToken</title>
+        </Head>
```
Comment (vpontis) [inline docs/pages/_app.tsx:68]: We can render this on the actual `index.tsx` page

**Review by vpontis — CHANGES_REQUESTED** [review]
Hmm, so this splash page doesn't feel like the rest of the docs. It feels like they are two different websites. 

It also doesn't say NFToken or have any information. I think we should have a more informative splash page — at least with a tagline — and have it _feel_ a bit more like the rest of the docs.

**Comment by benborgers** [comment]
@vpontis I’ve done another iteration! It felt a little empty on the right side so I added a sketch but we can totally replace that with something more professionally done. What do you think? What other info do you think belongs on the splash page? 

2022-06-14 at 12 23@2x

**Comment by vpontis** [comment]
I like this a lot more! Did you draw this? If so, could you make another one but w/ a square frame and a cartoon-ish ape in the frame? Haha...

Also let's make it — "A simple, inexpensive Solana NFT Standard"

**Comment by vpontis** [comment]
And instead of "Mint in your browser" → "Create an NFT"

**Comment by benborgers** [comment]
Sure! How about this?

2022-06-14 at 14 57@2x

**Review by vpontis — APPROVED** [review]
Sweet!

**Pull request merged**